spec/helpers/application_helper_spec.rb

```diff
@@ -41,7 +41,7 @@ def current_user
         AppConfig.privacy.jquery_cdn = true
       end
 
-      it 'inclues jquery.js from jquery cdn' do
+      it 'includes jquery.js from jquery cdn' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.